### PR TITLE
Update reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the instructions as it was confusing a little bit on which module to connect before running the cmdlets.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/958

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 1]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
 > [!NOTE]
 > This process also works for soft-deleted mailboxes if they connect to a new on-premises AD account. The `SoftDeletedMailbox` parameter should be used to replace the `InactiveMailboxOnly` parameter.
 
-1. Run the following command to obtain inactive mailbox attributes from Exchange:
+1. Run the following command to obtain inactive mailbox attributes from Exchange Online powershell:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:47]:
```suggestion
1. Run the following command to obtain inactive mailbox attributes from Exchange Online PowerShell:
```

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 2]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
 
     ```powershell
     $InactiveMailbox = Get-Mailbox -InactiveMailboxOnly -Identity <identity of inactive mailbox>
     ```
 
-2. Run the following command to temporarily associate the inactive mailbox with a cloud account. An account will be synced back to Azure AD:
+2. Run the following command to temporarily associate the inactive mailbox with a cloud account from Exchange online powershell.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:53]:
```suggestion
2. Run the following command to temporarily associate the inactive mailbox with a cloud account from Exchange Online PowerShell:
```

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 3]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
 
    ```powershell
    New-Mailbox -InactiveMailbox $InactiveMailbox.DistinguishedName -Name "<name of inactive mailbox>" -DisplayName "<DisplayName of inactive mailbox>" -MicrosoftOnlineServicesID <alias@*.onmicrosoft.com> -Password (ConvertTo-SecureString -String <PasswordString> -AsPlainText -Force) -ResetPasswordOnNextLogon $true
    ```
 
-3. Obtain the **ImmutableID** parameter value. By default, this is the on-premises `ObjectGUID` attribute as a base-64 string. You can convert `ObjectGUID` by using the following command in Windows PowerShell:
+3. Connect to the exchange management shell on your on premise server and collect the `ObjectGUID` attribute.   You can convert `ObjectGUID` to 'ImmutableID' by using the following command in Windows PowerShell:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:59]:
```suggestion
3. Connect to the exchange management shell on your on premise server and collect the `ObjectGUID` attribute.   You can convert `ObjectGUID` to 'ImmutableID' by using the following command in Windows PowerShell:
```
```suggestion
3. Connect to the Exchange Management Shell on your on-premises server and collect the `ObjectGUID` attribute. You can convert `ObjectGUID` to 'ImmutableID' by using the following command in Windows PowerShell:
```

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 4]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
-4. In Azure AD, obtain the `ObjectID` parameter value for new Azure AD account that was synced back from Exchange Online. To do this, run the following command:
-
-   ```powershell
-   Get-MsolUser -UserPrincipalName <UPN> | fl objectID
-   ```
-
-5. Set the `ImmutableID` parameter in Azure AD:
+  Obtain the immutable ID.
+  
+4. Connect to Azure AD and  Set the `ImmutableID` parameter in Azure AD :
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:66]:
```suggestion
4. Connect to Azure AD and Set the `ImmutableID` parameter in Azure AD :
```

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 5]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
 
     ```powershell
-    Set-MsolUser -ObjectId  <ObjectId>  -ImmutableId <ImmutableId>
+    Set-MsolUser -userprincipalname  -ImmutableId <ImmutableId>
     ```
 
-6. Bring the original Azure AD account into the scope of Azure AD Connect.
+5. Perform a Delta sync on AD Connect.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:72]:
```suggestion
5. Perform a Delta sync on Azure AD Connect.
```

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 6]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
+
+6.  This connects original Azure AD account into the scope of Azure AD Connect
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:74]:
```suggestion
6.  This connects the original Azure AD account into the scope of Azure AD Connect 
```

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 7]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
    ```
-
-4. In Azure AD, obtain the `ObjectID` parameter value for new Azure AD account that was synced back from Exchange Online. To do this, run the following command:
-
-   ```powershell
-   Get-MsolUser -UserPrincipalName <UPN> | fl objectID
-   ```
-
-5. Set the `ImmutableID` parameter in Azure AD:
+  Obtain the immutable ID.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:64]:
```suggestion
  4. Obtain the immutable ID.
```

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 8]
```diff
@@ -44,37 +44,34 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
-    Set-MsolUser -ObjectId  <ObjectId>  -ImmutableId <ImmutableId>
+    Set-MsolUser -userprincipalname  -ImmutableId <ImmutableId>
     ```
 
-6. Bring the original Azure AD account into the scope of Azure AD Connect.
+5. Perform a Delta sync on AD Connect.
+
+6.  This connects original Azure AD account into the scope of Azure AD Connect 
 
 7. Check the mailbox object, and verify that the primary SMTP address is updated from a temporary user principal name (UPN) value to the correct primary address.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:76]:
```suggestion
8. Check the mailbox object, and verify that the primary SMTP address is updated from a temporary user principal name (UPN) value to the correct primary address.
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:76]: Please also change the numbering of the current line 82 to "9".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 9]
```diff
@@ -44,45 +44,42 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
 
-2. Run the following command to temporarily associate the inactive mailbox with a cloud account. An account will be synced back to Azure AD:
+2. Run the following command to temporarily associate the inactive mailbox with a cloud account from Exchange Online PowerShell:
 
    ```powershell
    New-Mailbox -InactiveMailbox $InactiveMailbox.DistinguishedName -Name "<name of inactive mailbox>" -DisplayName "<DisplayName of inactive mailbox>" -MicrosoftOnlineServicesID <alias@*.onmicrosoft.com> -Password (ConvertTo-SecureString -String <PasswordString> -AsPlainText -Force) -ResetPasswordOnNextLogon $true
    ```
 
-3. Obtain the **ImmutableID** parameter value. By default, this is the on-premises `ObjectGUID` attribute as a base-64 string. You can convert `ObjectGUID` by using the following command in Windows PowerShell:
+3. Connect to the exchange management shell on your on premise server and collect the `ObjectGUID` attribute.   You can convert `ObjectGUID` to `ImmutableID` by using the following command in Windows PowerShell:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:59]:
```suggestion
3. Connect to the exchange management shell on your on-premises server and collect the `ObjectGUID` attribute.   You can convert `ObjectGUID` to `ImmutableID` by using the following command in Windows PowerShell:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by helenclu** [comment]
@msbemba , could you add the command for this step:
Connect to the exchange management shell on your on-premises server and collect the `ObjectGUID` attribute.

**Comment by msbemba** [comment]
@helenclu I have added the steps to collect the object ID, I have used the Get-aduser instead of a get-remotemailbox (in exchange) to reduce ambiguity

File: Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md [hunk 10]
```diff
@@ -44,45 +44,45 @@ Restore the content from the inactive mailbox to the newly provisioned mailbox b
...
-5. Set the `ImmutableID` parameter in Azure AD:
+5. Set the `ImmutableID` parameter in Azure AD :
 
     ```powershell
-    Set-MsolUser -ObjectId  <ObjectId>  -ImmutableId <ImmutableId>
+    Set-MsolUser -UserPrincipalName <UPN> -ImmutableId <ImmutableId>
     ```
 
-6. Bring the original Azure AD account into the scope of Azure AD Connect.
+6. Run a Azure AD Connect delta sync. This brings the original Azure AD account into the scope of Azure AD Connect.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/administration/reconnect-inactive-or-soft-deleted-mailboxes-to-ad.md:77]:
```suggestion
6. Run an Azure AD Connect delta sync. This brings the original Azure AD account into the scope of Azure AD Connect.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba - Please also commit this minor suggestion. Many thanks.

**Review by ninobb — APPROVED** [review]

**Pull request merged**